slm_lab/env/vizdoom/__init__.py

```diff
@@ -0,0 +1 @@
+from .vizdoom_env import VizDoomEnv
```

Comment:
Great so far. NOTE: code review cutoff; long series of config files begin below.